modules/apps/document-library/document-library-external-video/src/main/resources/META-INF/resources/js/ExternalVideoPreview.js

```diff
@@ -92,28 +92,32 @@ const ExternalVideoPreview = ({
 					{Liferay.Language.get('video-url-help')}
 				</p>
 
-				{HTML ? (
-					<div
-						className="file-picker-preview-video"
-						dangerouslySetInnerHTML={{__html: HTML}}
-					/>
-				) : (
-					<div className="file-picker-preview-video">
-						<div className="file-picker-preview-video-placeholder">
-							{loading ? (
-								<ClayLoadingIndicator />
-							) : (
-								<ClayIcon symbol="video" />
-							)}
+				<div className="file-picker-preview-video">
+					{HTML ? (
```

Comment:
Can't we change this variable name to something more clear? `previewHtml`, `videoHtml`, ... 
Also, notice that we should use lowerCase